jasmine/dependencies.bzl

```diff
@@ -14,36 +13,28 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 # changes in this function should be marked as BREAKING in the commit message
 # and released only in semver majors.
 def rules_jasmine_dependencies():
-    # The minimal version of bazel_skylib we require
-    maybe(
-        http_archive,
+    http_archive(
         name = "bazel_skylib",
-        sha256 = "c6966ec828da198c5d9adbaa94c05e3a1c7f21bd012a0b29ba8ddbccb2c93b0d",
-        urls = [
-            "https://github.com/bazelbuild/bazel-skylib/releases/download/1.1.1/bazel-skylib-1.1.1.tar.gz",
-            "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.1.1/bazel-skylib-1.1.1.tar.gz",
```

Comment:
Was there any reason to have a mirror before?

Reply:
The bazel build mirror urls is just copy pasted around. githbu.com is sufficient and adding the bazel mirror as a fallback doesn't add much value imo.